### PR TITLE
command/exec: Add unit tests

### PR DESCRIPTION
**Pull request by vadorovsky**
Signed-off-by: Michal Rostecki <mrostecki@suse.de>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/6137)

### CODE REVIEW

**Comment by vadorovsky** [comment]
test-me-please

**Review by eloycoto — APPROVED** [review]

**Comment by vadorovsky** [comment]
test-me-please

File: pkg/command/exec/exec.go [hunk 1]
```diff
@@ -54,16 +54,6 @@ func combinedOutput(ctx context.Context, cmd *exec.Cmd, filters []string, scoped
 	return out, err
 }
 
-// CombinedOutput runs the command 'cmd' which was initialized with reference
-// to 'ctx', and logs an error to 'scopedLog', with more verbosity if 'verbose'
-// is set to true.
-//
-// Returns any error (including timeout) that may have occurred during
-// execution of 'cmd'.
-func CombinedOutput(ctx context.Context, cmd *exec.Cmd, scopedLog *logrus.Entry, verbose bool) ([]byte, error) {
```
Comment (vadorovsky) [inline pkg/command/exec/exec.go:63]: This function is not used anywhere. Which is probably good - that means that `CombinedOutput` method of `Cmd` struct is used everywhere.

Comment (vadorovsky) [reply pkg/command/exec/exec.go:63]: @joestringer Do you agree with removing it?

**Review by joestringer — APPROVED** [review]
Good catch, and agreed the static function should not have been used. LGTM.

**Comment by ianvernon** [comment]
Test failure is unrelated: 
https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/7790/

**Comment by ianvernon** [comment]
Merging, as this only adds unit tests and removes a function which is dead code. Thanks @mrostecki !

**Pull request merged**